person.rb

```diff
@@ -1,10 +1,13 @@
-# class Person
+require_relative 'nameable'
+# class Person < Nameable
 class Person
```

Comment:
- [ ] Kindly remove commented code

- [ ] The Person class should inherit from Nameable class

  ```
  class Person < Nameable
  ```

- [ ] There should be a `correct_name` method in the Person class.

  ```
  def correct_name
    @name
  end
  ```